api/serverV2.go

```diff
@@ -76,9 +76,10 @@ func NewServerV2(
 	bfIndexer blockindex.BloomFilterIndexer,
 	actPool actpool.ActPool,
 	registry *protocol.Registry,
+	workerNumbers int,
```

Comment:
Let's wrap this params in `opts`
```
func WithWorkers() Option {}
```
The default number of workers in API config is 5. If the number needs to be changed,  WithWorkers() can be used then.